dashboard/src/t5gweb/t5gweb.py

```diff
@@ -196,6 +198,9 @@ def init_cache():
     if bugs is None:
         logging.warning("no bugs found in cache. refreshing...")
         libtelco5g.cache_bz(cfg)
+    if escalations is None:
```

Comment:
the updated cache_cards function pulls escalations so it should be cached first. I rebuilt in my dev environment w/ empty cache and errored out:
```
2022-03-11T20:14:09.578153855+00:00 stderr F   File "/srv/t5gweb/t5gweb.py", line 197, in init_cache
2022-03-11T20:14:09.578226181+00:00 stderr F     libtelco5g.cache_cards(cfg)
2022-03-11T20:14:09.578226181+00:00 stderr F   File "/srv/t5gweb/libtelco5g.py", line 711, in cache_cards
2022-03-11T20:14:09.578386639+00:00 stderr F     if case_number in escalations:
2022-03-11T20:14:09.578401309+00:00 stderr F TypeError: argument of type 'NoneType' is not iterable
```